code/_onclick/ai.dm

```diff
@@ -187,3 +189,8 @@
 
 	else if(down)
 		TD.move_camera_by_click()
+
+/turf/AICtrlClick(mob/living/silicon/ai/user)
+	var/turf/T = get_turf(src)
```

Comment:
I dont event know what get turf is doing on a turf.... maybe it gravs the area

Anyway you just want src, you do not need loc or that line